doc/extensions.md

```diff
@@ -0,0 +1,18 @@
+# Extending Files
+
+Here you will find information on how you can tune the behaviour of certain parts
+of the library.
+
+## Disabling file existance checks in the FileManager during file loading
```

Comment:
```suggestion
## Disabling file existance checks in the FileManager during entities' hydration
```

Reply:
You can load a file without hydrating an entity.